browser/lib/findNoteTitle.js

```diff
@@ -1,8 +1,18 @@
+const frontMatterRegex = /^\-{3,}/
```

Comment:
> The front matter must be the first thing in the file and must take the form of valid YAML set between triple-dashed lines. 

This regex will also valid for line that started with 4 dashes. You should remove the `,` there.

Reply:
Yes, you are right.

Reply:
I think we should stick to the traditional `---` :smile: